src/app/components/static/static.component.html

```diff
@@ -66,91 +18,57 @@ <h3 class="animated3">
-				</div>
-			</div>
-		</div>
-		<div class="row">
-			<div class="col-md-4 col-sm-6 col-xs-12">
-				<div class="blogList_single_post clear_fix wow fadeInUp animated" style="visibility: visible; animation-name: fadeInUp;">
-					<div class="img_holder">
-						<img [src]="img1" alt="News" class="img-responsive">
-					</div> <!-- End of .img_holder -->
-					<div class="post">
-						<div class="news-date">
-							August 20, 2017
-						</div>
-						<div class="text">
-							<p class="author"><a href="#">by Mark Vilton</a></p>
-							<h4><a href="#">Design and Advanced Industry Materials of Innovation</a></h4>
-							<p>The increasing availability of productivity <br>information provides an opportunity for the products materials. </p>
-							<div class="link"><a href="#" class="tran3s">Read more</a></div>
+					<div class="col-md-6 col-sm-12">
 							
-						</div>
-						
-					</div> <!-- End of .post -->
-				</div>
-				
-			</div>
-			<div class="col-md-4 col-sm-6 col-xs-12">
-				<div class="blogList_single_post clear_fix wow fadeInUp animated" style="visibility: visible; animation-name: fadeInUp;">
-					<div class="img_holder">
-						<img [src]="img2" alt="News" class="img-responsive">
-					</div> <!-- End of .img_holder -->
-					<div class="post">
-						<div class="news-date">
-							August 25, 2017
-						</div>
-						<div class="text">
-							<p class="author"><a href="#">by Mark Thornton</a></p>
-							<h4><a href="#">Commercial and military space projects around the world.</a></h4>
-							<p>Our special grade steels are being used in commercial and military aerospace projects around the world.</p>
-							<div class="link"><a href="#" class="tran3s">Read more</a></div>
-							
-						</div>
-						
-					</div> <!-- End of .post -->
-				</div>
-				
-			</div>
-			<div class="col-md-4 col-sm-6 col-xs-12">
-				<div class="blogList_single_post clear_fix wow fadeInUp animated" style="visibility: visible; animation-name: fadeInUp;">
-					<div class="img_holder">
-						<img [src]="img3" alt="News" class="img-responsive">
-					</div> <!-- End of .img_holder -->
-					<div class="post">
-						<div class="news-date">
-							September 20, 2017
-						</div>
-						<div class="text">
-							<p class="author"><a href="#">by Mark Vilton</a></p>
-							<h4><a href="#">Design and Advanced Industry Materials of Innovation</a></h4>
-							<p>The increasing availability of productivity <br>information provides an opportunity for the products materials.  </p>
-							<div class="link"><a href="#" class="tran3s">Read more</a></div>
-							
-						</div>
-						
-					</div> <!-- End of .post -->
-				</div>
-				
+							<figure class="about-img">
+									<img [src]="aboutImg" alt="about photo" width="570" height="360">
+							</figure>
+					</div>
+					<div class="col-md-6 col-sm-12">
+							<div class="about-text">
```

Comment:
Spacing is too huge here, please make it readable (4px/level) and try to keep lines shorten then 120px